packages/opentelemetry-metrics/src/Metric.ts

```diff
@@ -133,7 +133,7 @@ export class CounterMetric extends Metric<BoundCounter> implements api.Counter {
    * @param labels key-values pairs that are associated with a specific metric
```

Comment:
the doc should probably look like this after the param is optional
`* @param [labels = {}] key-values pairs ........`

Reply:
Thank you.  This docblock's type has been changed per the your correction.